### ISSUE TITLE
Should upspin-ui wizard allow entering an existing proquint?

### ISSUE THREAD

**Issue by addos** [upspin/upspin#609]
Hi, when re-installing/setting up upspin, is there anyway to have upspin-ui prompt you for an existing proquint and regenerate private/public keys based on the proquint entered? This would make it much easier to re-install, re-bootstrap a new system using existing keys by allowing you to use the proquint it asks you to keep safe.

**Comment by adg** [comment]
Yeah this makes a lot of sense. It would be a good addition to the upspin-ui setup wizard.
